internal/store/bundle.json

```diff
@@ -0,0 +1 @@
+{"description":"","invocationImages":null,"name":"bundle-name","schemaVersion":"","version":""}
```

Comment:
What's this?

Reply:
seems some temporary version of bundle_test did write to `.` and I committed by mistake